src/libraries/CurrencyRatioSortOrder.sol

```diff
@@ -0,0 +1,13 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity ^0.8.20;
+
+library CurrencyRatioSortOrder {
+    int256 constant NUMERATOR_MOST = 300;
+    int256 constant NUMERATOR_MORE = 200;
+    int256 constant NUMERATOR = 100;
+
+    int256 constant DENOMINATOR_MOST = -300;
+    int256 constant DENOMINATOR_MORE = -200;
+    int256 constant DENOMINATOR_2 = -150;
```

Comment:
but there also could be the native token wrapped too, like WMATIC...
the more im thinking about it, the more i think it would be cleaner to just remove the priority and not care about wrapped native tokens...

Reply:
and then that way wrappedNative and native have the same priority. 
we would then not care about WETH priority on those chains.
so on those chains it could be possible to display price like WETH/CELO

Reply:
mm thats true ok. I just hate the name `DENOMINATOR_2` 😂 please can you rename it? then we can keep it hahah

Reply:
oh i was thinking we could just remove it altogether